scripts/core/kernel.yml

```
@@ -358,6 +358,16 @@ params:
 returns:
     - $X_RESULT_ERROR_INVALID_KERNEL_ARGUMENT_INDEX
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "Properties for for $xKernelSetArgMemObj."
+class: $xKernel
+name: $x_mem_obj_properties_t
```

I missed this before, I think we should namespace this struct to mention kernel arg so its obviously tightly coupled to `urKernelSetArgMemObj`:

```suggestion
name: $x_kernel_arg_mem_obj_properties_t
```